pkg/ebpf/c/stdarg.h

```diff
@@ -1,4 +1,3 @@
-R"***(
 /*===---- stdarg.h - Variable argument handling ----------------------------===
```

Comment:
I don't think we should leave this file in the `pkg/ebpf/c` since those files are meant to be used directly by ebpf programs. Being underneath `pkg/ebpf/compiler` makes more sense to me.

We also have this file in-tree, as well as a version coming from S3. Since this file is Apache2 licensed, we should be able to embed it. Could we try using `go:embed` and then writing it to a temp file during compilation? That would be one less file we have to worry about in our packaging.

Reply:
that's a great idea!